CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 Note: This is in reverse chronological order, so newer entries are added to the top.
 
+Swift v.Next
+-----------
+* [#2937]
+   * Improvements
+   
+   Adding a dependency requirement can now be done with the convenience initializer `.package(url: , branch:)`
```

Comment:
Couple minor edits: we want to use the correct PR number and write change details as punctuated sentences. 😉

```suggestion
* [#3292]
   Adding a dependency requirement can now be done with the convenience initializer `.package(url: , branch:)`.
```

Reply:
Oh I see what happened, I wanted make sure the styling was the same so I just copied a section then proceeded to never update the PR number 😅.